vtex/billing-app
# Use user-provided app name to install app

#### What is the purpose of this pull request?
To use user-provided app name to install app

#### What problem is this solving?
Trying to install an app by specifying its version would not give the expected result. This was caused by the use of the last published manifest to create the appId used to install the app. Now, all the interpretation of the user-provided app name is given to Apps.

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in your workspace
2. Install any app by specifying its version, for example:
```
vtex install vtex.sms-provider@0.4.1
```
3. The version installed should be the one you specified

#### Screenshots or example usage
None to show

#### Types of changes
